pkgs/applications/networking/msmtp/paths.patch

```diff
@@ -1,55 +1,59 @@
+commit 35cab741af069571cf4c55e0ce1ae96617d5778c
+Author: Peter Hoeg <peter@hoeg.com>
+Date:   Fri Aug 12 10:52:22 2022 +0800
+
+    nixify
+
```

Comment:
Did you mean to include the commit information in the diff this time?

Reply:
Nope, I did not. A consequence of using `git show` rather than `git diff`. Fixed.